Cargo.toml

```diff
@@ -14,7 +14,7 @@ members = ["tools/*"]
 rgb = "0.8.33"
 image = "0.24.2"
 anyhow = "1.0.58"
-cgmath = { version = "0.18.0", features = ["mint"] }
+glam = { version = "0.20", features = ["mint"] }
```

Comment:
Since `bevy_math` re-exports glam, this can be safely removed.

Reply:
Actually, I had to add it with the `mint` feature for converting the point data from fbxcel-dom, because the bevy glam dep isn't compiled with `mint`. It's not a major feature, I could just edit the code where the fbxcel-dom `PointX<foo>` are converted into bevy_math `FooVecX`.

Reply:
I will PR `mint` feature to `bevy_math`, so we possibly will get it in 0.8 to not duplicate same dep only to add feature and possibly future authors of various plugins might benefit from this change.

Reply:
As long as the version for glam here matches the version Bevy uses it won't duplicate the dependency.

Reply:
Good to know, thanks